hera_cal/omni.py

```diff
@@ -1297,7 +1300,10 @@ def omni_apply(files, opts):
             for bl, k in zip(*np.unique(mir.baseline_array, return_index=True)):
                 blmask = np.where(mir.baseline_array == bl)[0]
                 ai, aj = mir.baseline_to_antnums(bl)
-                if not ai in cal.ant_array or not aj in cal.ant_array:
+                if ai not in cal.ant_array or aj not in cal.ant_array:
+                    if opts.flag_missing:
```

Comment:
Should it not be an option to flag missing antennas, but a necessity? I'm more in favor of having this be the default behavior of applying cal solutions. At the very least have the options default be True.

Reply:
That's a good point. I'll make the option default to True.